CHANGELOG.md

```diff
@@ -22,7 +22,9 @@
 
 ##### Bug Fixes
 
-* None.
+* Fix TypeError - no implicit conversion of nil into String.
```

Comment:
this should say what the underlying bug is, rather than the exception being avoided